<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bluetooth</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system-bluetooth"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bluetooth</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="paneSelection">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="panes">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="devicesList"/>
          </item>
          <item>
           <widget class="QFrame" name="devControlFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pairButton">
               <property name="text">
                <string>Pair</string>
               </property>
               <property name="icon">
                <iconset theme="list-add"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeButton">
               <property name="text">
                <string>Remove</string>
               </property>
               <property name="icon">
                <iconset theme="edit-delete"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="connectButton">
               <property name="text">
                <string>Connect</string>
               </property>
               <property name="icon">
                <iconset theme="network-connect"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="disconnectButton">
               <property name="text">
                <string>Disconnect</string>
               </property>
               <property name="icon">
                <iconset theme="network-disconnect"/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="scanButton">
               <property name="text">
                <string>Scan</string>
               </property>
               <property name="icon">
                <iconset theme="view-refresh"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="visibilityBox">
            <property name="text">
             <string>Device Visible to other devices</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
